PositiveAirwayPressureDevices/R4/resources/Questionnaire-R4-PositiveAirwayPressureDevicesFaceToFace.json

```diff
@@ -547,1028 +298,99 @@
-          "valueCodeableConcept": {
-            "coding": [
-              {
-                "system": "http://hl7.org/fhir/questionnaire-item-control",
-                "code": "gtable"
-              }
-            ]
-          }
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/review-of-system"
         }
       ],
-      "linkId": "10",
+      "type": "display",
+      "text": "This is a placeholder for Review Of System"
+    },
+    {
+      "linkId": "11",
       "type": "group",
-      "repeats": true,
-      "text": "Medication",
+      "text": "Physical Examination With Vital Signs",
       "item": [
         {
-          "linkId": "10.1",
-          "text": "RxNorm",
-          "type": "string"
-        },
-        {
-          "linkId": "10.2",
-          "text": "Description",
-          "type": "string"
-        },
-        {
-          "linkId": "10.3",
-          "text": "Dose",
-          "type": "string"
-        },
-        {
-          "linkId": "10.4",
-          "text": "Frequency",
-          "type": "string"
-        },
-        {
-          "linkId": "10.5",
-          "text": "Route",
-          "type": "string"
-        },
-        {
-          "linkId": "10.6",
-          "text": "Status",
-          "type": "choice",
-          "answerOption": [
-            {
-              "valueCoding": {
-                "display": "New"
-              }
-            },
-            {
-              "valueCoding": {
-                "display": "Current"
-              }
-            },
+          "linkId": "11.1",
+          "type": "group",
+          "text": "Vital Sign",
+          "item": [
             {
-              "valueCoding": {
-                "display": "Modified"
-              }
+              "extension": [
+                {
+                  "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+                  "valueCanonical": "questionnaire/vital-signs"
+                },
+                {
+                  "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire-expand",
+                  "valueBoolean": true
+                }
+              ],
+              "linkId": "11.1.1",
+              "type": "display",
+              "text": "This is a placeholder for Vital Signs"
             },
             {
-              "valueCoding": {
-                "display": "Discontinued"
-              }
+              "required": true,
+              "linkId": "11.1.2",
+              "text": "Neck circumference (cm)",
+              "extension": [
+                {
+                  "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+                  "valueExpression": {
+                    "language": "text/cql",
+                    "expression": "\"RespiratoryAssistDevicesFaceToFacePrepopulation\".NeckCircumf"
+                  }
+                }
+              ],
```

Comment:
it is better to add 'qestionnaireUnit' extension with 'cm'

Reply:
The expression is coded incorrectly with "RespiratoryAssistDevicesFaceToFacePrepopulation"; it should be "PositiveAirwayPressureDevicePrepopulation".

Reply:
Since physical exam is supposed to be performed during the evaluation and we are not supposed to pre-populate the fields from EHR. I removed it from the questionnaire.